packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -402,15 +407,27 @@ export class AadAppForTeamsImpl {
 
   private static getRedirectUris(
     frontendEndpoint: string | undefined,
-    botEndpoint: string | undefined
+    botEndpoint: string | undefined,
+    clientId: string
   ) {
-    const redirectUris: string[] = [];
+    const redirectUris: IAADDefinition = {
+      web: {
+        redirectUris: [],
+      },
+      spa: {
+        redirectUris: [],
+      },
+    };
     if (frontendEndpoint) {
-      redirectUris.push(`${frontendEndpoint}/auth-end.html`);
+      redirectUris.web?.redirectUris?.push(`${frontendEndpoint}/auth-end.html`);
+      redirectUris.spa?.redirectUris?.push(`${frontendEndpoint}/auth-end.html/blank.html`);
```

Comment:
appending a page after another page makes the url seems strange, can we remove `auth-end.html` and give the blank page a more meaningful name?

Reply:
Good idea. Will update, thx！

Reply:
Updated.